auth/src/main/resources/rmap-schema-derby.sql

```diff
@@ -1,46 +0,0 @@
-CREATE TABLE ApiKeys (
-  apiKeyId INT NOT NULL GENERATED BY DEFAULT AS IDENTITY,
-  accessKey VARCHAR(256) DEFAULT NULL,
-  secret VARCHAR(1024) DEFAULT NULL,
-  keyUri VARCHAR(128) DEFAULT NULL,
-  label VARCHAR(128) DEFAULT NULL,
-  note VARCHAR(1024) DEFAULT NULL,
-  keyStatus VARCHAR(32) DEFAULT NULL,
-  includeInEvent BOOLEAN DEFAULT NULL,
-  startDate TIMESTAMP DEFAULT NULL,
-  endDate TIMESTAMP DEFAULT NULL,
-  createdDate TIMESTAMP DEFAULT NULL,
-  lastModifiedDate TIMESTAMP DEFAULT NULL,
-  revokedDate TIMESTAMP DEFAULT NULL,
-  userId INT DEFAULT NULL,
-  PRIMARY KEY (apiKeyId)
-);
-
-CREATE TABLE UserIdentityProviders (
-  userIdentityProviderId INT NOT NULL GENERATED BY DEFAULT AS IDENTITY,
-  identityProvider VARCHAR(128) DEFAULT NULL,
-  providerAccountId VARCHAR(256) DEFAULT NULL,
-  providerAccountPublicId VARCHAR(45) DEFAULT NULL,
-  providerAccountDisplayName VARCHAR(45) DEFAULT NULL,
-  providerAccountProfileUrl VARCHAR(512) DEFAULT NULL,
-  lastAuthenticatedDate TIMESTAMP DEFAULT NULL,
-  createdDate TIMESTAMP DEFAULT NULL,
-  userId INT NOT NULL,
-  PRIMARY KEY (userIdentityProviderId)
-);
-
-
-CREATE TABLE Users (
-  userId INT NOT NULL GENERATED BY DEFAULT AS IDENTITY,
-  name VARCHAR(256) NOT NULL,
-  email VARCHAR(128) NOT NULL,
-  isActive BOOLEAN NOT NULL,
-  rmapAgentUri VARCHAR(128) DEFAULT NULL,
-  authKeyUri VARCHAR(256) DEFAULT NULL,
-  createdDate TIMESTAMP NOT NULL,
-  lastAccessedDate TIMESTAMP DEFAULT NULL,
-  cancellationDate TIMESTAMP DEFAULT NULL,
-  rmapDiSCOUri VARCHAR(45) DEFAULT NULL,
-  doRMapAgentSync BOOLEAN NOT NULL,
-  PRIMARY KEY (userId)
-);
```

Comment:
Sure we can leave it. Or copy and paste it to documentation, but yes, it isn't used by anything.  If it is useful for documentation, I can copy and paste it somewhere if you'd like.

Reply:
Maybe we can just leave the createTables.sql one where it is instead of deleting it, what do you think? https://github.com/rmap-project/rmap/tree/master/auth

Reply:
maybe just exclude this commit: https://github.com/rmap-project/rmap/pull/210/commits/85de3c32b5bf4262ba55c2140fa6122bb89f2846

Reply:
don't really mind where it goes, just not an obvious home for it on the wiki yet, so perhaps it should stay with the github project

Reply:
I dropped the commit!